From 4c45770160c05dd16cacfb5e9910b452d340fc71 Mon Sep 17 00:00:00 2001
From: Samreen Nilofer <samreen.nilofer@intel.com>
Date: Thu, 12 Feb 2015 10:03:02 +0530
Subject: [PATCH 17/24] ASoC: dfw: Add media2_out capture pipeline

Issue: GMINL-4319

Added new media2_out pipeline in media task for capture path.
This pipeline will gets the data from interleaver and will be connected to frontend.
media2_out will be connected to primary capture path.

Change-Id: I56ba8469ad6e2b938b8a8348c7f2da78404f08e9
Signed-off-by: Samreen Nilofer <samreen.nilofer@intel.com>
Signed-off-by: Praveen Diwakar <praveen.diwakar@intel.com>
---
 sound/soc/intel/platform-libs/atom_pipes.h | 29 +++++++++++++++--------------
 1 file changed, 15 insertions(+), 14 deletions(-)

diff --git a/sound/soc/intel/platform-libs/atom_pipes.h b/sound/soc/intel/platform-libs/atom_pipes.h
index 47178a0..c5b445e 100644
--- a/sound/soc/intel/platform-libs/atom_pipes.h
+++ b/sound/soc/intel/platform-libs/atom_pipes.h
@@ -53,24 +53,25 @@ enum sst_dfw_path_index {
 	SST_DFW_PATH_INDEX_PCM0_OUT	      = (0x0D << SST_DFW_PATH_ID_SHIFT),
 	SST_DFW_PATH_INDEX_PCM1_OUT	      = (0x0E << SST_DFW_PATH_ID_SHIFT),
 	SST_DFW_PATH_INDEX_PCM2_OUT	      = (0x0F << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_AWARE_OUT	      = (0x10 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_VAD_OUT	      = (0x11 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_AWARE_OUT	      = (0x12 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_VAD_OUT	      = (0x13 << SST_DFW_PATH_ID_SHIFT),
 
-	SST_DFW_PATH_INDEX_MEDIA0_OUT	      = (0x12 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_MEDIA1_OUT	      = (0x13 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_MEDIA0_OUT	      = (0x14 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_MEDIA1_OUT	      = (0x15 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_MEDIA2_OUT	      = (0x16 << SST_DFW_PATH_ID_SHIFT),
 
-	SST_DFW_PATH_INDEX_FM_OUT	      = (0x14 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_FM_OUT	      = (0x18 << SST_DFW_PATH_ID_SHIFT),
 
-	SST_DFW_PATH_INDEX_PROBE1_PIPE_OUT    = (0x15 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE2_PIPE_OUT    = (0x16 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE3_PIPE_OUT    = (0x17 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE4_PIPE_OUT    = (0x18 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE5_PIPE_OUT    = (0x19 << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE6_PIPE_OUT    = (0x1A << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE7_PIPE_OUT    = (0x1B << SST_DFW_PATH_ID_SHIFT),
-	SST_DFW_PATH_INDEX_PROBE8_PIPE_OUT    = (0x1C << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE1_PIPE_OUT    = (0x19 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE2_PIPE_OUT    = (0x1A << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE3_PIPE_OUT    = (0x1B << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE4_PIPE_OUT    = (0x1C << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE5_PIPE_OUT    = (0x1D << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE6_PIPE_OUT    = (0x1E << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE7_PIPE_OUT    = (0x1F << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_PROBE8_PIPE_OUT    = (0x20 << SST_DFW_PATH_ID_SHIFT),
 
-	SST_DFW_PATH_INDEX_SIDETONE_OUT	      = (0x1D << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_SIDETONE_OUT	      = (0x21 << SST_DFW_PATH_ID_SHIFT),
 
 	/* Start of input paths */
 	SST_DFW_PATH_INDEX_MODEM_IN	      = (0x80 << SST_DFW_PATH_ID_SHIFT),
-- 
1.9.1

